sdk/src/main/scala/com/horizen/account/state/AccountStateView.scala

```diff
@@ -331,12 +361,15 @@ class AccountStateView(private val metadataStorageView: AccountStateMetadataStor
     stateDb.getNonce(address)
   }
 
-  override def getAccountStateRoot: Array[Byte] = metadataStorageView.getAccountStateRoot
+  override def getLogs(txHash: Array[Byte]): Array[EvmLog] = {
+    stateDb.getLogs(txHash)
```

Comment:
I think, to make it work we need to implement `setupTxContext` that will call stateDB.txLookup() method.
Otherwise, statedb doesn't know how to store the logs.

Reply:
Ok, I added the implementation. I also added the txIndex to the signature since evm needs it too